packages/next-app/src/components/Table.tsx

```diff
@@ -3,6 +3,23 @@ import { createContext, useContext } from "react";
 
 const TableContext = createContext({});
 
+const predefinedClasses = {
+  gray: {
+    solid: {
+      dark: "bg-gray-800 border border-gray-700",
```

Comment:
Same as [this](https://github.com/bleu-studio/balancer-pool-metadata/pull/84/files#r1185004036). Let's use only one source for the color palette definition throughout these apps -- we should probably even add a lint rule to disallow tw colors or remove them altogether if we can -- feel free to log this as a task or take a stab at doing it if you think the time is right

Reply:
Sure! [Here](https://linear.app/bleu-llc/issue/BAL-332/adapt-colors-to-use-only-radixs-palette) 

Also, @mendesfabio could you change the colors on figma to match [Radix's colors](https://www.radix-ui.com/colors)?